### ISSUE TITLE
All styles except full & header silently swallow binary data

### ISSUE THREAD

**Issue by lewisthompson** [sharkdp/bat#466]
```
% bat --version
bat 0.9.0
% bat --style=plain .DS_Store
% bat --style=changes .DS_Store
% bat --style=grid .DS_Store
% bat --style=numbers .DS_Store
% bat --style=header .DS_Store
File: .DS_Store   <BINARY>
```

`--style=full` omitted as it's somewhat bulky.

Personally I am using `bat` indirectly through commands I'm already familiar with:

```
% which cat
cat: aliased to bat --plain --paging=never
% which more
more: aliased to bat --plain --pager="less --no-init --quit-at-eof"
```

So my preference would be to just spew binary data and junk to my terminal.  However, in other modes any sort of indication that the output is being omitted would be helpful.

I believe this is related to https://github.com/sharkdp/bat/issues/248 and the issue is present in `master`.

**Comment by sharkdp** [comment]
Thank you very much for the feedback.

> However, in other modes any sort of indication that the output is being omitted would be helpful.

We should probably fix this, yes. I'd say we print a warning to stderr ("Omitting binary file ...") if the `--style` does not include a header line.

> my preference would be to just spew binary data and junk to my terminal

We should probably add a flag to allow for this. Currently there is no way to force `bat` to print files which it detected as "binary" (see [this article](https://dev.to/sharkdp/what-is-a-binary-file-2cf5) for more information on how `bat` does this).

**Comment by sharkdp** [comment]
Closed via #550 

See also #530

**Comment by sharkdp** [comment]
Fixed in [v0.11.0](https://github.com/sharkdp/bat/releases/tag/v0.11.0).
